src/app/common/elements/datepicker.tsx

```diff
@@ -315,51 +319,74 @@ const DatePicker: React.FC<DatePickerProps> = ({ selectedDate, format = "MM/DD/Y
         }
     };
 
-    const handleKeyDown = (event, currentPart) => {
-        const key = event.key;
+    const handleKeyDown = (event, currentPart) => {};
 
-        if (key === "ArrowLeft" || key === "ArrowRight") {
-            // Handle arrow navigation without selecting text
-            handleArrowNavigation(event, currentPart);
-            return;
-        }
+    const handleFocus = (event, part) => {
+        event.target.select();
```

Comment:
currently when you select the date picker for the first time, handleFocus gets called twice and so the keys get registered twice. I think that's because handleMouseDown calls handleFocus which calls event.target.select() which I think triggers handleFocus again ? 

Either way it's not a big deal and doesn't affect anything. I could in theory add an mobx observable here so that they don't register twice. like keybindsRegistered: OV<boolean> which then gets set to false when the keybinds get unregistered

I think this is worth doing because there's a log print that happens if a key bind gets registered twice

Reply:
addressed - I implemented it as described